src/app/components/Home.js

```diff
@@ -265,7 +265,7 @@ class HomeComponent extends Component {
         if (user.is_admin) {
           return true;
         }
-        const teams = JSON.parse(user.user_teams);
+        const teams = typeof user?.user_teams === 'string' ? JSON.parse(user.user_teams) : {};
```

Comment:
why not just use our `safelyParseJSON` helper?

Reply:
Sorry @amoedoamorim,  I forgot we already have this helper. I've just pushed the update :)